skimage/measure/fit.py

```diff
@@ -680,6 +845,39 @@ def _dynamic_max_trials(n_inliers, n_samples, min_samples, probability):
     return np.ceil(np.log(nom) / np.log(denom))
 
 
+def add_from_estimate(cls):
+    """Add ``from_estimate`` method  class using ``estimate`` method"""
+
+    if hasattr(cls, 'from_estimate'):
+        if not ismethod(cls.from_estimate):
+            raise TypeError(f'Class {cls} `from_estimate` must be a ' 'class method.')
+        return cls
+
+    if not hasattr(cls, 'estimate'):
+        raise TypeError(
+            f'Class {cls} must have `from_estimate` class method '
+            'or `estimate` method.'
+        )
+
+    warn(
+        "Passing custom classes without `from_estimate` has been deprecated "
+        "since version 0.26 and will be removed in version 2.2. "
```

Comment:
Version check here too.

Reply:
Do you mean - printed version should be 2.0 here - or something else?

Reply:
Assuming you are referring to version being 2.0 or 2.2 - and that we've resolved this in the meeting discussion.